geth/api/api.go

```diff
@@ -188,3 +194,33 @@ func (api *StatusAPI) JailCall(chatID, this, args string) string {
 func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
+
+// TODO(oskarth): API package this stuff
+func (api *StatusAPI) Notify(token string) string {
+	log.Debug("Notify", "token", token)
+
+	var NP fcm.NotificationPayload
+	NP.Title = "Status - new message"
+	NP.Body = "ping"
+
+	// TODO(oskarth): Experiment with this
+	data := map[string]string{
```

Comment:
Do we want to leave this unchangable ?

Reply:
@influx6 I'd say say, because then we can see what type of information we can pass through on client side. For example, hiding encrypted payload etc.

Reply:
Alright. 👍